### PR TITLE
Playground compat: stop using `bool` db field type

### PR DESCRIPTION
**Pull request by imath**
Updates BuddyPress db tables schema so that it uses tinyint fields instead of boolean ones. This is required to have WP Playground work as expected.

NB: this move is requiring a major version to be achieved.

Trac ticket: https://buddypress.trac.wordpress.org/ticket/9126

---
**This Pull Request is for code review only. Please keep all other discussion in the BuddyPress Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the WordPress Core Handbook for more details.**

### CODE REVIEW

File: blueprint.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"landingPage": "\/wp-admin\/admin.php?page=bp-components",
+	"preferredVersions": {
+		"php": "8.1",
```
Comment (renatonascalves) [inline blueprint.json:5]: Not PHP 8.2 or PHP 8.3?

File: blueprint.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+		"wp": "6.5"
```
Comment (renatonascalves) [inline blueprint.json:6]: Is it possible to set it to `latest`?

File: blueprint.json (hunk 1 above)
Comment (imath) [reply blueprint.json:5]: Good point!
```suggestion
		"php": "8.2",
```

File: blueprint.json (hunk 2 above)
Comment (imath) [reply blueprint.json:6]: It is! Just checked: https://wordpress.github.io/wordpress-playground/blueprints-api/index/
```suggestion
		"wp": "latest"
```

**Pull request closed without merging**